### ISSUE TITLE
Add Liblog and add log statements

### ISSUE THREAD

**Issue by abergs** [abergs/fido2-net-lib#48]
LibLog: https://github.com/damianh/LibLog

And we should add error logs when running without MDS.

**Comment by iamcarbon** [comment]
@abergs This library is deprecated. Do we still have a need for any logging functionality? Can we close this issue?

**Comment by abergs** [comment]
Yes it seems the community now have moved towards https://www.nuget.org/packages/Microsoft.Extensions.Logging.Abstractions

I still think it's valuable for consumers of the library if they can get insight into how the library executed for a specific result.
Right now it would only be via sourcelink.

Keeping this open.
